lib/utility.js

```diff
@@ -0,0 +1,7 @@
+const Utils = {
+  slugify: function(str) {
+    return str.replace(/[^A-Za-z0-9]/g, `-`).toLowerCase();
```

Comment:
@Pomax ah k, I guess I will need to find a new package to do this then. Cuz `slug` package produced various errors when I deployed the MozFest site to Heroku.
<img width="1183" alt="screen shot 2017-05-24 at 10 42 18 am" src="https://cloud.githubusercontent.com/assets/2896608/26462453/b45e2fc2-4135-11e7-87cc-128258819b94.png">